migration/migration2.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+import (
+	"database/sql"
+)
+
+var mig2 = Migration{
+	StepUp: func(tx *sql.Tx) error {
+		_, err := tx.Exec(`
+			CREATE TABLE rule (
+				module          VARCHAR PRIMARY KEY,
```

Comment:
module is fqdn right? TPB their repository is a bit inconsistent in this case :)

Reply:
I used the names from the YAML files.

Reply:
Also not sure about the `status`, but it was on the whiteboard, so I wrote it down that way.

Reply:
ok agreed